### PR TITLE
Add Dockerfile with MongoDB

### PR DESCRIPTION
**Pull request by rodrigok**
This will generate a new Docker image, in addition to the existent ones, for preview and tests purposes.

### CODE REVIEW

**Comment by geekgonecrazy** [comment]
Are we sure we want to do this?  I think this will be a support nightmare.  Not to mention hurting the docker purist in me so very badly.  Multiple processes in a docker container kind of defeats the whole purpose of docker and containerization.

File: .docker-mongo/entrypoint.sh [hunk 1]
```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+mongod --fork --config /etc/mongod.conf
```
Comment (geekgonecrazy) [inline .docker-mongo/entrypoint.sh]: even though not meant for production and only preview... Maybe we should add: `--smallfiles --storageEngine=mmapv1`

For sure I think the storageEngine bit.

**Comment by toughIQ** [comment]
I am also more the „one container, one process“ guy. What does the addition of mongo solve or improve? I would stick with the docker-compose way of combining Rocket.Chat and mongo containers.

**Comment by geekgonecrazy** [comment]
I just edited my reply.  But talked a bit, the goal is for quick and easy way to **preview** Rocket.Chat on-premises.  Also for use in testing.  For sure not an alternative way to run a production system.

**Comment by engelgabriel** [comment]
Can we add a README.MD and a BIG comment on the docker file that this is for preview and automated testing only, and not meant for production environments?

**Comment by geekgonecrazy** [comment]
Yes this would help a ton!  I wonder if we want to make it super easy... Should we go so far as to set it up with a user so they just login and start using?  Helping re-inforce(hopefully) that it's a preview?

**Comment by JSzaszvari** [comment]
@engelgabriel is on the money here. 

There really needs to be a large warning everywhere possible that this is for previewing and not production. 

Might be worth putting this in the container as a start up banner similar to what I've done for the rocket.chat hubot container 

https://user-images.githubusercontent.com/872463/40983464-0a38b460-6923-11e8-8f50-79aca6e50859.png

**Comment by engelgabriel** [comment]
@geekgonecrazy can you add this sweet ascii text to `.docker-mongo/entrypoint.sh`?

**Comment by rodrigok** [comment]
We need to define if this will be a tag like `rocketchat:0.61.0-preview` or if this will be another image (I prefer this) like `rocketchat-preview:0.61.0`

**Comment by geekgonecrazy** [comment]
@JSzaszvari what did you use to generate that text for hubot?  

@rodrigok I agree.  It should be a completely separate image. Treat it as completely separate thing so it's always clear it's not production with every little detail 😁

**Comment by sampaiodiego** [comment]
As per suggestions, I've [added](https://github.com/RocketChat/Rocket.Chat/pull/10971/files#diff-814d954e8b559e9db8b34258c8919151R3) the Rocket.Chat Preview banner.

I've also [included](https://github.com/RocketChat/Rocket.Chat/pull/10971/files#diff-1d37e48f9ceff6d8030570cd36286a61R325) the new docker image on CI as `rocket.chat.preview` .. it will have the same tags as the official one

**Pull request merged**